modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-service/service.xml

```diff
@@ -39,9 +39,14 @@
 		<finder name="CPDefinitionId" return-type="Collection">
 			<finder-column name="CPDefinitionId" />
 		</finder>
-		<finder name="Number" return-type="CPDefinitionDiagramEntry" unique="true">
+		<finder name="C_N" return-type="CPDefinitionDiagramEntry" unique="true">
```

Comment:
@igorbeslic @Commando18 If we want to be more specific to avoid future naming issues we can do `CPD_N`, does not look like we need to include the I for id.

Reply:
I did this to be consistent with all the other services we have. Even if we have companyId field we always used C_X_Y for CPDefinitionId field when possible. If I have to change it anyway, let me know. Thanks guys!

Reply:
@Commando18 I think its fine to leave it the way you have it for consistency.

Reply:
Ok, I can resolve this one as well then. Thanks!